packages/unlock-js/src/web3Service.ts

```diff
@@ -420,4 +420,29 @@ export default class Web3Service extends UnlockService {
     )
     return ethers.BigNumber.from(await lockContract.numberOfOwners()).toNumber()
   }
+
+  /**
+   * Returns total of key for a specific address
+   * @param {String} lockAddress
+   * @param {String} address
+   * @param {Number} network
+   */
+  async totalKeys(lockAddress: string, owner: string, network: number) {
+    const contract = await this.getLockContract(
+      lockAddress,
+      this.providerForNetwork(network)
+    )
+
+    // for locks >= v11
+    if (contract.totalKeys) {
```

Comment:
So instead of doing that, please add `totalKeys` for version 7 and for version 11 and have them call the right ABI function.
Look at how we do the same thing for other functions. 
This is a lot more robust.